docs/permissions/custom-rules.md

```diff
@@ -111,7 +118,31 @@ class TestPermissionPolicy implements PermissionPolicy {
 ...
 ```
 
+## Provide the rule during plugin setup
+
+Now that we have a custom rule defined and added to our policy, we need provide it to the catalog plugin. This step is important because the catalog plugin will use the rule's `toQuery` and `apply` methods while evaluating conditional authorize results. There's no guarantee that the catalog and permission backends are running on the same server, so we must explicitly link the rule to ensure that it's available at runtime.
+
+The api for providing custom rules may differ between plugins, but there should typically be some integration point during the creation of the backend router. For the catalog, this integration point is exposed via `CatalogBuilder.addPermissionRules`.
+
+```typescript title="packages/backend/src/plugins/catalog.ts"
+import { CatalogBuilder } from '@backstage/plugin-catalog-backend';
+/* highlight-add-next-line */
+import { isInSystemRule } from './permissions';
```

Comment:
Sadly we get a type error with this:

![image](https://github.com/backstage/backstage/assets/67169551/a76a3f11-ef2b-41d0-8c3e-3bd20fb91f58)